src/hooks/useTracking.js

```diff
@@ -100,7 +100,7 @@ export const useTracking = (
       /* eslint-enable camelcase */
     }
 
-    sendEvent(SCHEMA_NAME, SCHEMA_REV, language, event)
+    sendEvent(SCHEMA_NAME, STREAM_NAME, event)
```

Comment:
Shall we remove `language` as an argument (line 28)?

Reply:
Done